src/components/m-datepicker/js/datepicker.js

```diff
@@ -18,12 +18,12 @@ export default class Datepicker {
     this.datepickerBody = this.wcNode.querySelector('.js-datepicker__datepicker-body');
     this.dropdownMonth = this.wcNode.querySelector('.js-datepicker__dropdown__month');
     this.dropdownYear = this.wcNode.querySelector('.js-datepicker__dropdown__year');
-    this.okButton = this.wcNode.querySelector('.js-datepicker__button__Ok');
-    this.cancelButton = this.wcNode.querySelector('.js-datepicker__button__Cancel');
+    this.okButton = this.wcNode.querySelector('.js-datepicker__button-ok');
```

Comment:
definitely better, we should never have camel-case, upper-case in our class names ...

Reply:
I removed all datepicker changes. they were not meant to be in this PR.